server.py

```diff
@@ -61,6 +62,11 @@ def create_ffmpeg_stream(video_path:str, video_type:State, loop=False):
         stderr=subprocess.DEVNULL
     )
     process_dict[video_type] = process.pid
+    if video_type == State.PLAYING:
+        MetricsHandler.streams_count.labels(video_type="playing").inc(amount=1)
+    else:
+        MetricsHandler.streams_count.labels(video_type="interlude").inc(amount=1)
+
```

Comment:
i think this will do the same thing
```suggestion
    MetricsHandler.streams_count.labels(video_type=vide_type.value).inc()

```